frontend/dist/css/index.css

```diff
@@ -54,6 +71,13 @@ nav {
   font-size: 30px;
 }
 
+/*login btn*/
```

Comment:
This is an actual comment, right? Can we rewrite it so it looks like one? /* Login button for ### page. */